components/brave_wallet_ui/page/container.tsx

```diff
@@ -305,6 +301,16 @@ function Container (props: Props) {
     return formatPrices(grandTotal)
   }, [userAssetList])
 
+  const onChangeTimeline = (timeline: AssetPriceTimeframe) => {
+    if (selectedAsset) {
+      props.walletPageActions.selectAsset({ asset: selectedAsset, timeFrame: timeline })
+    } else {
+      if (fullPortfolioBalance !== '0.00') {
```

Comment:
It's better to cast this string to a float and then compare with `0`.

Reply:
Done!